core/sql/executor/ExHdfsScan.cpp

```diff
@@ -374,10 +375,9 @@ ex_tcb_private_state *ExHdfsScanTcb::allocatePstates(
 Int32 ExHdfsScanTcb::fixup()
 {
   lobGlob_ = NULL;
-
-  ExpLOBinterfaceInit
-    (lobGlob_, (NAHeap *)getGlobals()->getDefaultHeap(),getGlobals()->castToExExeStmtGlobals()->getContext(),TRUE, hdfsScanTdb().hostName_,hdfsScanTdb().port_);
-  
+  lobGlob_ = ExpLOBoper::initLOBglobal((NAHeap *)getGlobals()->getDefaultHeap(),
+                                         getGlobals()->castToExExeStmtGlobals()->getContext(), 
+                                         useLibhdfsScan_);
```

Comment:
To preserve the old hdfs scan functionality, we may need to preserve the starting of the threads so the scan works correctly when use_lib_hdfs is ON.

Reply:
Not sure what you meant. The threads are started if useLibHdfsScan_ is set true via the above method too

Reply:
Yes. Found the problem isHiveRead is not passed through the new methods. Fixing it to ensure that use_libhdfs 'on' works for hdfs scan